Python/Tests/TestAdapterTests/TestDiscovererTests.cs

```diff
@@ -462,6 +462,28 @@ public abstract partial class TestDiscovererTests {
             DiscoverTests(testEnv, new[] { testFilePath }, runSettings, expectedTests);
         }
 
+
+        [TestMethod, Priority(0)]
+        [TestCategory("10s")]
```

Comment:
What's this test category for?

Reply:
Steve was using this more consistently than I am to mark the tests that take a while